calculator.js

```diff
@@ -0,0 +1,207 @@
+var writes = document.getElementById('screen');
+var currNumber = 0;
+var result = 0;
+
+var buttons = document.querySelector('.allButtons');
+var numberButtons = buttons.querySelectorAll('.number');
+
+for (var i = 0; i <= numberButtons.length - 1; i++) {
+
+    numberButtons[i].onclick = function() {
+
+        var buttonNumber = parseInt(this.innerHTML);
+
+        currNumber = currNumber * 10 + buttonNumber
+
+        writes.value = currNumber;
+
+    }
+};
+
+var operationButtons = buttons.querySelectorAll('.operation');
+
+for (var i = 0; i <= operationButtons.length - 1; i++) {
+    var action = 1;
```

Comment:
Never use generic names, since they are hard to follow for rest of the team. Always use something which is easy to understand for anyone. In this case you do not even need a counter, a simple boolean will do the job:
```
var isInitialAction = false;
```